internal/testutils/broker.go

```diff
@@ -292,3 +292,13 @@ func parseSessionID(sessionID string) string {
 	}
 	return strings.TrimSuffix(cut[len(cut)-1], "-session_id")
 }
+
+// GenerateSessionID returns a sessionID that can be used in tests.
+func GenerateSessionID(username string) string {
+	return fmt.Sprintf("%s-session_id", username)
+}
+
+// GenerateEncryptionKey returns an encryption key that can be used in tests.
+func GenerateEncryptionKey(brokerName string) string {
+	return fmt.Sprintf("%s_key", brokerName)
```

Comment:
nipitck: -suffix in session id and _suffix for encryption key. Let’s ensure we use the same delimiter.

Reply:
Agreed. I've updated it to use a `-` for separating words and left the `_` to act like "spaces", which looks better IMHO.